src/utilities/modules/dispensary/constants.js

```diff
@@ -6,6 +6,7 @@
 export const CASH_TRANSACTION_KEYS = {
   NAME: 'name',
   TYPE: 'title',
+  PAYMENT_TYPE: 'description',
```

Comment:
`DESCRIPTION: 'description'`?

same with the one below, these constants are confusing

Reply:
Sorry, maybe need a comment. These are the keys for each cash transaction field, i.e. when rendering a `GenericChoiceList` of `PAYMENT_TYPE`, display the `description` field.

Reply:
3230de6.